modules/core/share_limits.py

```diff
@@ -185,7 +188,13 @@ def update_share_limits_for_group(self, group_name, group_config, torrents):
             torrent_upload_limit = -1 if round(torrent.up_limit / 1024) == 0 else round(torrent.up_limit / 1024)
             if group_config["limit_upload_speed"] == 0:
                 group_config["limit_upload_speed"] = -1
-            check_limit_upload_speed = group_config["limit_upload_speed"] != torrent_upload_limit
+            group_up_limit = (
+                round(group_config["group_upload_speed"] / len(torrents)) if group_config["group_upload_speed"] else None
```

Comment:
With the `default_is_none=True,` function in config.py we can simplify this and remove `if group_config["group_upload_speed"] else None`

Reply:
That sounds like an efficient change. Let's do it!